Use vtbackup in scheduled backups (#658)
planetscale/vitess-operator: docs/api.md

</td>
<td>
<em>(Optional)</em>
<p>A list of pointers to currently running jobs.

> This field is deprecated and no longer used in &gt;= v2.15. It will be removed in a future release.</p>

Reviewer: ```suggestion
This field is deprecated and no longer used in versions&gt;= v2.15. It will be removed in a future release.</p>
```

Author: fixed via 423e959

</td>
<td>
<em>(Optional)</em>
<p>Information when was the last time the job was successfully scheduled.
This field is deprecated and no longer used in &gt;= v2.15. It will be removed in a future release.
Please use lastScheduledTimes instead which maps the last schedule time to each VitessBackupScheduleStrategy
in the VitessBackupSchedule.</p>
</td>
</tr>
<tr>
<td>
<code>lastScheduledTimes</code><br>
<em>
<a href="https://v1-18.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#*k8s.io/apimachinery/pkg/apis/meta/v1.time--">
map[string]*k8s.io/apimachinery/pkg/apis/meta/v1.Time
</a>
</em>
</td>
<td>
<em>(Optional)</em>

> <p>LastScheduledTimes lists for every VitessBackupScheduleStrategy what is the last schedule we executed.

Reviewer: ```suggestion
<p>A list of the last schedule we executed for each VitessBackupScheduleStrategy. 
```

Author: fixed via 423e959

> It does not list when the last execution started, but to which scheduled time it corresponds.</p>

Reviewer: ```suggestion
Note that these are not the times when the last execution started, only the scheduled times.</p>
```

Author: fixed via 423e959

<td>
<p>Strategy defines how we are going to take a backup.
If you want to take several backups within the same schedule you can add more items
to the Strategy list. Each VitessBackupScheduleStrategy will be executed within different

> K8S Jobs. This is useful if you want to have a single schedule backing up multiple shards

Reviewer: ```suggestion
kubernetes jobs. This is useful if you want to have a single schedule backing up multiple shards
```

Author: fixed via 423e959

<em>(Optional)</em>
<p>ConcurrencyPolicy specifies ho to treat concurrent executions of a Job.
Valid values are:
- &ldquo;Allow&rdquo;: allows CronJobs to run concurrently;

> - &ldquo;Forbid&rdquo; (default): forbids concurrent runs, skipping next run if previous run hasn&rsquo;t finished yet;

Reviewer: 👍
